src/logics/Logic.cc

```diff
@@ -649,8 +649,11 @@ PTRef Logic::mkNot(PTRef arg) {
 PTRef Logic::mkConst(const char* name)
 {
     //assert(0);
-    //return PTRef_Undef;
-    assert(strlen(name) > 0);
+    // return PTRef_Undef;
+    if (strlen(name) == 0) {
```

Comment:
I don't think this is the right solution.

Looking at the code, I believe we can just remove this assert (and the commented-out code above).
`resolveTerm` will report an error in this situation.

In any case, OpenSMT does not support strings, so this input is not a valid input for OpenSMT anyway.